client/lib/abtest/active-tests.js

```diff
@@ -167,4 +167,13 @@ export default {
 		defaultVariation: 'control',
 		allowExistingUsers: true,
 	},
+	showCompositeCheckoutI18N: {
+		datestamp: '20200710',
+		variations: {
+			composite: 90,
+			regular: 10,
+		},
+		defaultVariation: 'regular',
+		allowExistingUsers: true,
```

Comment:
I _think_ that if you don't set `localeTargets`, you will only end up with EN language users, and we want other language users, right? Or is the intent to wait longer for the translations?

The README for `lib/abtest` reads

> * `localeTargets` - By default, tests only run on users where the locale is set to English. You can also run for all locales by setting `localeTargets` to 'any', or to an array of a specific locale or locales  `localeTargets: ['de']`